aana/utils/json.py

```diff
@@ -35,6 +35,8 @@ def json_serializer_default(obj: object) -> object:
         return obj.model_dump()
     if isinstance(obj, Path):
         return str(obj)
+    if isinstance(obj, type):
```

Comment:
Why do we need that?

Reply:
Converting Image calls to hash sometimes throw error for hashing the image_lib type which is a OPENCVWRAPPER. Adding this, fix it

Reply:
```
from pathlib import Path

from pydantic import BaseModel
from aana.core.models.image import Image
from aana.utils.core import get_object_hash
from aana.utils.json import jsonify

from aana.core.models.image_chat import ImageChatDialog
prompt = "Who is the painter of the image?"
image_path = "/root/aana_sdk/aana/tests/files/images/Starry_Night.jpeg"
image = Image(path=Path(image_path), save_on_disk=False, media_id="test_image")
dialog = ImageChatDialog.from_prompt(prompt=prompt, images=[image])
args_hash = jsonify(dialog)
```

Reply:
it happens when we want to to jsonify the ouput of a model_dumpp funtion which has type specification for attr.